tailwind.config.js

```diff
@@ -0,0 +1,109 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      borderWidth: {
+        sm: ".0938rem",
+      },
+      borderRadius: {
+        base: "1.1875rem",
+      },
+      boxShadow: {
+        pr: "1px 1px 14px rgba(0, 0, 0, 0.1)",
+      },
+      colors: {
+        primaryGray: {
+          50: "#E2E2E2",
+          100: "#EDEDED",
+          200: "#F4F4F4",
+          300: "#9A9A9A",
+          400: "#878787",
+        },
+        purple: "#5602E0",
+      },
+      fontFamily: {
+        neuf: ["NeufileGrotesk"],
```

Comment:
If you don't have a backup font, so you don't need an array here
```suggestion
        neuf: "NeufileGrotesk",
```